src/authorizations/actions/thunks.ts

```diff
@@ -168,6 +168,30 @@ export const deleteAuthorization = (id: string, name: string = '') => async (
   }
 }
 
+export const bulkDeleteAuthorizations = (tokenIds: string[]) => async (
+  dispatch: Dispatch<Action | NotificationAction>
+) => {
+  try {
+    for (const id of tokenIds) {
+      try {
+        const resp = await api.deleteAuthorization({authID: id})
```

Comment:
Totally, I'm not saying it won't work. The implementation doesn't take advantage of concurrent requests that browsers allow, hence one of the reasons behind the [no-await-in-loop](https://eslint.org/docs/rules/no-await-in-loop) lint rule. 

As an aside, doing this could trigger a heap of dispatches that can cause slowiness in the process. This same iteration & dispatch combo was previously used in the dashboard list page. One thing that could be done to improve performance here would be to reduce the number of dispatches to one by pushing in the successful IDs into an array, and dispatching a new event that would handle removing an array of values rather than dispatching multiple events.